additional_recipes/ros-humble-rmw-wasm-cpp/recipe.yaml

```diff
@@ -0,0 +1,96 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+package:
+  name: ros-humble-rmw-wasm-cpp
+  version: 1.1.0
+
+source:
+  git: https://github.com/Tobias-Fischer/rmw_wasm
+  branch: emscripten-async-fixes
+  target_directory: ros-humble-rmw-wasm-cpp/src/work
+
+build:
+  script:
+    build.sh
+    # sel(unix): build.sh
+    # sel(emscripten-wasm32): build.sh
+  number: 12
+
+
+about:
+  homepage: https://www.ros.org/
+  license: BSD-3-Clause
+  summary: |
+    Robot Operating System
+
+requirements:
+  build:
+    - if: target_platform == 'emscripten-wasm32'
+      then:
+        - emscripten_emscripten-wasm32
+      else:
+        - ${{compiler('c')}}
+        - ${{compiler('cxx')}}
+    - if: linux64
+      then:
+        - sysroot_linux-64 2.17
+    - if: unix
+      then:
+        - make
+        - coreutils
+    - if: osx
+      then: tapi
+    - if: build_platform != target_platform
+      then:
+        - pkg-config
+        - python
+        - cross-python_${{target_platform}}
+        - cython
+        - numpy
+    - ninja
+    - setuptools
+    - cmake
+
+  host:
+    - numpy
+    - pip
+    - if: build_platform == target_platform
+      then:
+        - pkg-config
+    - python
+    - ros-humble-rcutils
+    - ros-humble-rcpputils
+    - ros-humble-rmw
+    - ros-humble-wasm-cpp
+    - ros-humble-ament-cmake
+    - dynmsg
+    - ros-humble-rosidl-runtime-c
+    - ros-humble-rosidl-runtime-cpp
+    - ros-humble-rosidl-typesupport-introspection-c
+    - ros-humble-rosidl-typesupport-introspection-cpp
+    - ros-humble-yaml-cpp-vendor
+    - ros2-distro-mutex 0.6.* humble_*
```

Comment:
I am a bit afraid of all these hardcoded `ros2-distro-mutex` versions, but to be honest I do not have a quick solution for this.

Reply:
Awesome, thanks :)!